### ISSUE TITLE
`cargo nextest --version`'s output seems wrong?

### ISSUE THREAD

**Issue by eval-exec** [nextest-rs/nextest#1189]
```bash
$ cargo nextest --version
cargo-nextest-nextest 0.9.66
$ ~/.cargo/bin/cargo-nextest --version
cargo-nextest 0.9.66
```

The output of `cargo nextest --version` is `cargo-nextest-nextest 0.9.66`, I think this is not right?

**Comment by sunshowers** [comment]
This is an artifact of how `cargo-nextest` is invoked: via a `cargo-nextest` binary that has a `nextest` subcommand. I guess it's worth fixing.

**Comment by sunshowers** [comment]
This is now fixed as of 0.9.79:

```
% cargo nextest -V
cargo-nextest 0.9.79 (ec7861ab7 2024-10-02)

% cargo nextest --version
cargo-nextest 0.9.79 (ec7861ab7 2024-10-02)
release: 0.9.79
commit-hash: ec7861ab7c27a63f036eca5fb3bbe67bbe1c0dd1
commit-date: 2024-10-02
host: x86_64-unknown-linux-gnu
```
